### PR TITLE
[BFN] Fix syncd RPC compilation

### PR DESCRIPTION
**Pull request by msosyak**
**- What I did**
Fixed compilation with `ENABLE_SYNCD_RPC=y`

Need https://github.com/Azure/sonic-buildimage/pull/4253 as well

**- How I did it**

**- How to verify it**
Build with `ENABLE_SYNCD_RPC=y`

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by msosyak** [comment]
@qiluo-msft, @kcudnik Please review

File: platform/barefoot/rules.mk [hunk 1]
```diff
@@ -20,7 +20,12 @@ SONIC_ALL += $(SONIC_ONE_IMAGE) $(SONIC_ONE_ABOOT) \
              $(DOCKER_FPM)
 
 # Inject sai into sairedis
-$(LIBSAIREDIS)_DEPENDS += $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)  #$(LIBSAITHRIFT_DEV_BFN)
+$(LIBSAIREDIS)_DEPENDS += $(BFN_SAI) $(WNC_OSW1800_PLATFORM) $(BFN_INGRASYS_PLATFORM) $(BFN_PLATFORM)
+
+ifeq ($(ENABLE_SYNCD_RPC),y)
+$(LIBSAIREDIS)_DEPENDS += $(LIBSAITHRIFT_DEV)
+endif
+
```
Comment (qiluo-msft) [inline platform/barefoot/rules.mk]: Remove this empty line?

Comment (msosyak) [reply platform/barefoot/rules.mk]: Done

**Comment by qiluo-msft** [comment]
Retest vsimage please

**Comment by msosyak** [comment]
retest vsimage please

**Comment by msosyak** [comment]
retest vs please

**Comment by msosyak** [comment]
retest this please

**Comment by msosyak** [comment]
retest default please

**Comment by msosyak** [comment]
retest default please

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest this please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest vsimage please

**Comment by akokhan** [comment]
retest vsimage please

**Pull request merged**